src/ecalc/libraries/libecalc/common/libecalc/core/models/compressor/train/fluid.py

```diff
@@ -218,12 +217,12 @@ def mix_in_stream(
 
         other_fluid = NeqsimFluid.create_thermo_system(
             composition=other_fluid_stream.fluid_model.composition,
-            temperature_kelvin=other_fluid_stream.temperature_kelvin,
-            pressure_bara=other_fluid_stream.pressure_bara,
+            temperature_kelvin=temperature_kelvin,
```

Comment:
so, now we are basically saying that the temp and pressure for both streams is the same, even though it has been specified differently for the 2nd stream...how can this be right? A comment in code why this is correct?

Got this verified by @Jfoul , if he knows?

Reply:
never mind, saw wrong :p

Reply:
but, should there be a check that the streams has same perssure and temperature? even if that is not used any more?

Reply:
NeqSim will complain about it. But we might not want to rely on NeqSims Error messages?

Reply:
For the mixing of the streams it is quite important that they have the same pressure (else in reality you would have flow with the difference in pressure). It is not true that they need to have the same temperature though (this will cause no issue), so the more accurately the combined temperature should be used (an estimation is a weighted average of temperature with mass rate of the streams - not entirely accurate though, but more so than having the same temperature)

Reply:
I would want us to do this check yes, but might be out of scope of this issue, so ok to leave for another PR. Perhaps there are other checks we should do as well?

With regards to the temperature, that is currently not being used. Also, the args are misleading wrt. that pressure and temp must be the same, which is currently not tested.